docs/components/clerk-provider.mdx

```diff
@@ -81,80 +85,17 @@ ReactDOM.render(
 
 ## Props

```

Comment:
Are all of these optional? Or just some?
Because I believe they are all optional, and in that case, we can remove the `?` from each prop, and add copy that says:

```suggestion
## Props

All props below are optional.
```

Reply:
Some of them are required, most notably the `publishableKey` or `frontendApi` value